pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix

```diff
@@ -68,6 +68,8 @@ in stdenv.mkDerivation {
 
   inherit src;
 
+  nativeBuildInputs = [ gtk3 wrapGAppsHook ];
```

Comment:
Actually, `gtk3` should not be needed.

Reply:
I thought the same thing, but the error still appears on my machine if `gtk3` is not in `nativeBuildInputs`.

Perhap's it's because I'm not using Gnome DM?

Reply:
Weird. `wrapGAppsHook` should bring GTK already. Though if you need it here, move it to `buildInputs`. Just the hook is native.